ui/index.d.ts

```diff
@@ -498,9 +498,10 @@ export type CommandCallback = (context: IActionContext, ...args: any[]) => any;
 /**
  * Used to register VSCode commands. It wraps your callback with consistent error and telemetry handling
  * Use debounce property if you need a delay between clicks for this particular command
+ * The telemetry event for this command will be named telemetryId if specified, otherwise it defaults to the commandId
  * NOTE: If the environment variable `DEBUGTELEMETRY` is set to a non-empty, non-zero value, then telemetry will not be sent. If the value is 'verbose' or 'v', telemetry will be displayed in the console window.
  */
-export declare function registerCommand(commandId: string, callback: CommandCallback, debounce?: number): void;
+export declare function registerCommand(commandId: string, callback: CommandCallback, debounce?: number, telemetryId?: string): void;
```

Comment:
For my own education, can you share more about the use case for this?

Reply:
We don't want "bicep.showVisualizer"/etc to be the name of the telemetry - our PM wants it something like "command.showVisualizer" so we can easily sort/query on just commands.